ui/packages/catalog-ui-search/src/main/webapp/component/location-new/utils/dms-utils.js

```diff
@@ -264,8 +268,82 @@ function validateDms(dms) {
     return { valid: valid, error: error };
 }
 
+/*
+ *  Decimal degrees -> DMS conversion utils
+ */
+function ddToDmsCoordinateLat(dd, secondsPrecision = DEFAULT_SECONDS_PRECISION) {
+    if (isNaN(dd)) {
+        return undefined;
+    }
+
+    const direction = dd >= 0 ? 'N' : 'S';
+    return ddToDmsCoordinate(dd, direction, LAT_DEGREES_DIGITS, secondsPrecision);
+}
+
+function ddToDmsCoordinateLon(dd, secondsPrecision = DEFAULT_SECONDS_PRECISION) {
+    if (isNaN(dd)) {
+        return undefined;
+    }
+
+    const direction = dd >= 0 ? 'E' : 'W';
+    return ddToDmsCoordinate(dd, direction, LON_DEGREES_DIGITS, secondsPrecision);
+}
+
+function ddToDmsCoordinate(dd, direction, degreesPad, secondsPrecision = DEFAULT_SECONDS_PRECISION) {
+    const ddAbsoluteValue = Math.abs(dd);
+    const degrees = Math.trunc(ddAbsoluteValue);
+    const degreeFraction = ddAbsoluteValue - degrees;
+    const minutes = Math.trunc(60 * degreeFraction);
+    const seconds = 3600 * degreeFraction - 60 * minutes;
+    const secondsRounded = roundTo(seconds, secondsPrecision);
+    return {
+        coordinate: `${pad(degrees, degreesPad)}°${pad(minutes, 2)}'${padDecimal(secondsRounded, 2)}"`,
+        direction: direction
+    };
+}
+
+function roundTo(num, sigDigits) {
```

Comment:
✏️ There's a [toFixed()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed) method which does the same

Reply:
See my response to your first comment.